test/EndToEnd/tests/PackageNameSpaceTests.ps1

```diff
@@ -1,5 +1,6 @@
 function Test-PackageSourceMappingRestore-WithSingleFeed
 {
+    [SkipTest('https://github.com/NuGet/Client.Engineering/issues/1826')]
```

Comment:
Since our (test) source is public, I prefer to link to a public issue, so any external contributors interested in looking can find more information about why the test was skipped.

Reply:
I found a bunch of Client Engineering issue links used to skip tests in the repo, so figured the rule of "if it's not product specific, it goes in Client Engineering" won.

Reply:
Maybe we need to clarify it somewhere, but we have been/supposed to be tracking skipped tests in the public repo for some time now. 

I'm sure it's just one of those things that slips when reviewing most of the time.

Reply:
I'll allow you 2 to reach a resolution you find appropriate for this PR, but in general I think we should consider moving all test disabled issues to the Home repo.

Reply:
https://github.com/NuGet/Client.Engineering/issues/1827